packages/fxa-auth-server/lib/senders/emails/templates/verifySecondary/index.stories.ts

```diff
@@ -12,7 +12,7 @@ export default {
 
 const createStory = storyWithProps(
   'verifySecondary',
-  'Sent to verify the addition of a secondary email via link.',
+  '*NO LONGER USED* This used to be sent to verify the addition of a secondary email via link, but we only send a code now. This should be removed in the future.',
```

Comment:
We have a few of these now (I think only one so far that's already been converted), they're crossed out in the QA checklist sheet. I'll file an additional maintenance issue filed to remove associated files / remove references to these in the codebase. We might want one issue per task though, not sure, but I'll get it in a ticket either way.